package.json

```diff
@@ -18,12 +18,15 @@
   "contributors": [],
   "license": "OGL-UK-3.0",
   "dependencies": {
+    "@azure/identity": "^4.5.0",
```

Comment:
we normally fix packages to a specific version so can remove the prefixing `^` as this will get the latest version